src/main/java/com/synopsys/integration/jenkins/scan/global/ExceptionMessages.java

```diff
@@ -19,4 +21,15 @@ public static String scannerFailureMessage(String message) {
         return "Synopsys Security Scan failed!! " + message;
     }
 
+    public static Map<Integer, String> bridgeErrorMessages() {
+        Map<Integer, String> exitCodeToMessage = new HashMap<>();
+
+        exitCodeToMessage.put(1, "Workflow failed! Exit Code: 1 Undefined error, check error logs");
+        exitCodeToMessage.put(2, "Workflow failed! Exit Code: 2 Error from adapter");
+        exitCodeToMessage.put(3, "Workflow failed! Exit Code: 3 Failed to shutdown the Bridge");
+        exitCodeToMessage.put(8, "Workflow failed! Exit Code: 8 The config option bridge.break has been set to true");
+        exitCodeToMessage.put(9, "Workflow failed! Exit Code: 9 Bridge initialization failed");
+
```

Comment:
@kishorikumar and @sig-tithi 
These are the error messages for the known exceptions. Please check if these are the messages we want to show.